src/Compilers/VisualBasic/Test/Syntax/Parser/ParseLanguageVersionTests.vb

```diff
@@ -317,5 +317,41 @@ End Module
             Diagnostic(ERRID.ERR_LanguageVersion, "$""hello {x1}""").WithArguments("12.0", "interpolated strings").WithLocation(5, 18))
     End Sub
 
+    <Fact>
+    Public Sub TupleExpression()
+        Dim x = Nothing
+        ParseAndVerify(
+        <![CDATA[
+Module Module1
+    Function M() As String
+        Dim x1 = (1, 2)
+        Dim x2 = (A:=1, B:=2)
 
+        Return nothing
+    End Function
+End Module
+        ]]>.Value,
+            LanguageVersion.VisualBasic14,
+            Diagnostic(ERRID.ERR_LanguageVersion, "(1, 2)").WithArguments("14.0", "tuples").WithLocation(4, 18),
+            Diagnostic(ERRID.ERR_LanguageVersion, "(A:=1, B:=2)").WithArguments("14.0", "tuples").WithLocation(5, 18))
+    End Sub
+
+    <Fact>
+    Public Sub TupleType()
+        Dim x = Nothing
+        ParseAndVerify(
+        <![CDATA[
+Module Module1
+    Function M() As String
+        Dim x1 As (Integer, Integer) = Nothing
+        Dim x1 As (A As Integer, B As Integer) = Nothing
+
+        Return Nothing
+    End Function
+End Module
+        ]]>.Value,
+            LanguageVersion.VisualBasic12,
```

Comment:
Anything not 15 would do, but 14 seems more appropriate.

---
In reply to: [71416864](https://github.com/dotnet/roslyn/pull/12596#discussion_r71416864) [](ancestors = 71416864)